package.json

```diff
@@ -33,13 +33,13 @@
     "react-static-container": "^1.0.0-alpha.1"
   },
   "peerDependencies": {
+    "babel-relay-plugin": "^0.3.0",
     "react": "^0.14.0",
     "react-dom": "^0.14.0"
   },
   "devDependencies": {
     "babel-core": "^5.8.25",
     "babel-eslint": "^4.1.1",
-    "babel-relay-plugin": "^0.3.0",
```

Comment:
You should probably keep it here too - in npm3 peer deps don't get installed by `npm install`.